data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -1,101 +1,25 @@
-            "Percent of individuals < 200% Federal Poverty Line (percentile)"
-        )
-
-        self.POVERTY_LESS_THAN_100_FPL_FIELD: str = (
-            "Percent of individuals < 100% Federal Poverty Line (percentile)"
-        )
-
-        # FEMA Risk Index
-        self.NATIONAL_RISK_FIELD: str = (
-            "FEMA Risk Index Expected Annual Loss Score (percentile)"
-        )
-
-        # DOE energy burden
-        self.ENERGY_BURDEN_FIELD: str = "Energy burden (percentile)"
-
-        # Diesel particulate matter
-        self.DIESEL_FIELD: str = "Diesel particulate matter (percentile)"
-
-        # PM2.5
-        self.PM25_FIELD: str = "Particulate matter (PM2.5) (percentile)"
-
-        # Traffic proximity and volume
-        self.TRAFFIC_FIELD: str = "Traffic proximity and volume (percentile)"
-
-        # Lead paint
-        self.LEAD_PAINT_FIELD: str = (
-            "Percent pre-1960s housing (lead paint indicator) (percentile)"
-        )
-
-        # Housing cost burden
-        self.HOUSING_BURDEN_FIELD: str = "Housing burden (percent) (percentile)"
-
-        # Wastewater discharge
-        self.WASTEWATER_FIELD: str = "Wastewater discharge (percentile)"
-
-        # Diabetes
-        self.DIABETES_FIELD: str = (
-            "Diagnosed diabetes among adults aged >=18 years (percentile)"
-        )
-
-        # Asthma
-        self.ASTHMA_FIELD: str = (
-            "Current asthma among adults aged >=18 years (percentile)"
-        )
-
-        # Heart disease
-        self.HEART_DISEASE_FIELD: str = (
-            "Coronary heart disease among adults aged >=18 years (percentile)"
-        )
-
-        # Life expectancy
-        self.LIFE_EXPECTANCY_FIELD: str = "Life expectancy (years) (percentile)"
-
-        # Unemployment
-        self.UNEMPLOYMENT_FIELD: str = (
-            "Unemployed civilians (percent) (percentile)"
-        )
-
-        # Median income as % of AMI
-        self.MEDIAN_INCOME_FIELD: str = (
-            "Median household income (% of AMI) (percentile)"
-        )
-
-        # Linguistic isolation
-        self.LINGUISTIC_ISO_FIELD: str = (
-            "Linguistic isolation (percent) (percentile)"
-        )
-
-        # Less than high school education
-        self.HIGH_SCHOOL_ED_FIELD: str = "Percent individuals age 25 or over with less than high school degree (percentile)"
-
-        # Set thresholds for score L
+class ScoreL(Score):
+    def __init__(self, df: pd.DataFrame) -> None:
         self.LOW_INCOME_THRESHOLD: float = 0.60
         self.ENVIRONMENTAL_BURDEN_THRESHOLD: float = 0.90
+        super().__init__(df)
 
-    def add_definition_l_factors(self):
-        self.df["Climate Factor (Definition L)"] = self.climate_factor()
-        self.df["Energy Factor (Definition L)"] = self.energy_factor()
+    def add_columns(self) -> pd.DataFrame:
+        logger.info("Adding Score L")
+
+        self.df["Climate Factor (Definition L)"] = self._climate_factor()
```

Comment:
I'm wondering if these names should also go in the `field_names` constants?

Reply:
done!